library/std/src/os/rtems/mod.rs

```diff
@@ -0,0 +1,4 @@
+#![stable(feature = "raw_ext", since = "1.1.0")]
```

Comment:
```suggestion
#![stable(feature = "raw_ext", since = "1.1.0")]
#![forbid(unsafe_op_in_unsafe_fn)]
```

We have been adding this forbid in `os` and `pal` modules in preparation of e2024.